pkg/agent/util/net_windows.go

```diff
@@ -357,9 +372,10 @@ func ConfigureLinkAddresses(idx int, ipNets []*net.IPNet) error {
 
 // PrepareHNSNetwork creates HNS Network for containers.
 func PrepareHNSNetwork(subnetCIDR *net.IPNet, nodeIPNet *net.IPNet, uplinkAdapter *net.Interface) error {
+	klog.InfoS("Creating HNSNetwork", "name", LocalHNSNetwork, "subnet", subnetCIDR, "nodeIP", nodeIPNet, "adapter", uplinkAdapter)
```

Comment:
should we start using structured logging for all new code?

Reply:
Do you see any guide or convention for this? I'm not sure as it seems K8s still mixes both usages. The difference I can tell is the structured logging message will always be quoted:
```
I0526 07:05:24.722781       1 gc_controller.go:161] GC'ing orphaned
I0526 07:05:24.722814       1 gc_controller.go:224] GC'ing unscheduled pods which are terminating.
I0526 07:05:32.162653       1 deployment_controller.go:176] "Updating deployment" deployment="dev1/nginx"
I0526 07:05:32.162747       1 deployment_controller.go:576] "Started syncing deployment" deployment="dev1/nginx" startTime="2021-05-26 07:05:32.16269763 +0000 UTC m=+116.896274878"
I0526 07:05:32.225902       1 deployment_controller.go:281] "ReplicaSet updated" replicaSet="dev1/nginx-64f6956798"
I0526 07:05:32.225941       1 replica_set.go:321] ReplicaSet nginx-64f6956798 updated. Desired pod count change: 10->8
I0526 07:05:32.225970       1 event.go:291] "Event occurred" object="dev1/nginx" kind="Deployment" apiVersion="apps/v1" type="Normal" reason="ScalingReplicaSet" message="Scaled down replica set nginx-64f6956798 to 8"
I0526 07:05:32.225995       1 controller_utils.go:206] Controller dev1/nginx-64f6956798 either never recorded expectations, or the ttl expired.
I0526 07:05:32.226154       1 replica_set.go:595] "Too many replicas" replicaSet="dev1/nginx-64f6956798" need=8 deleting=2
```
I would prefer using structured logging when there are arguments but not sure if it's better to use it for single message without arguments. Guess there would be lots of review efforts to unify it if it's not a common sense yet.

Reply:
I like the automatic quoting :). I don't think we need to go back and update the whole code base, but we can start using it for new code. Our CONTRIBUTING guidelines already points to the K8s guide, which now only mentions structured logging: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md

I believe that for a single message with not arguments, there is no difference between `InfoS` and `Infof` / `Info`.

Reply:
Without arguments, the record printed by `InfoS` will still be quoted while the one by `Info` will not be quoted.
```
I0608 13:11:17.829711   74722 main.go:22] "hello world with InfoS"
I0608 13:11:17.829863   74722 main.go:23] hello world with Infof
I0608 13:11:17.829870   74722 main.go:24] hello world with Info
```